bindings/nodejs/examples/how_tos/accounts_and_addresses/create-address.ts

```diff
@@ -26,9 +26,10 @@ async function run() {
         // To create an address we need to unlock stronghold.
         await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
 
-        const address = (await account.generateEd25519Addresses(1))[0];
+        const accountAddress = (await account.generateEd25519Addresses(1))[0];
+        const address = Utils.parseBech32Address(accountAddress.address);
 
-        console.log(`Generated address:`, address.address);
+        console.log(`Generated address:`, address);
```

Comment:
Shouldn't we show the bech32 address here?

Reply:
this will print:
```
Generated address: Ed25519Address {
  type: 0,
  pubKeyHash: '0x2435ee1fdc75f8fc8d3bcfa7140d334bbe378a1280423bff584989a92b341bbe'
}
```

Do you wnat it toprint the whole `AccountAddress`?

Reply:
No no, just address.address as it was before?

Reply:
In the python example also only print the bech32 address

Reply:
The toString() only prints the bech32 address or the full account address now?

Reply:
the address, see here: https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/lib/types/block/address.ts#L46C15-L46C15

Reply:
But this is for `Address` and the function returns `AccountAddress`?